workbench/src/renderer/components/PluginModal/index.jsx

```diff
@@ -29,36 +34,88 @@ export default function PluginModal(props) {
       if (addPluginErr) {
         setErr(addPluginErr);
       } else {
-        setShowAddPluginModal(false);
+        setShowPluginModal(false);
       }
     });
   };
-  const handleChange = (event) => {
-    setURL(event.currentTarget.value);
+
+  const removePlugin = () => {
+    setLoading(true);
+    ipcRenderer.invoke(ipcMainChannels.REMOVE_PLUGIN, pluginToRemove).then(() => {
+      updateInvestList();
+      setLoading(false);
+      setShowPluginModal(false);
+    });
   };
 
+  useEffect(() => {
+    ipcRenderer.invoke(ipcMainChannels.GET_SETTING, 'plugins').then(
+      (data) => {
+        setPlugins(data);
+        setPluginToRemove(Object.keys(data)[0]);
+      }
+    );
+  }, [loading]);
+
   const { t } = useTranslation();
 
   let modalBody = (
     <Modal.Body>
       <Form>
         <Form.Group className="mb-3">
-          <Form.Label htmlFor="url">Git URL</Form.Label>
+          <Form.Label htmlFor="url">Add a plugin</Form.Label>
           <Form.Control
             id="url"
             name="url"
             type="text"
             placeholder={t('Enter Git URL')}
-            onChange={handleChange}
+            onChange={(event) => setURL(event.currentTarget.value)}
           />
+          <Form.Text className="text-muted">
+            {t('This may take several minutes')}
+          </Form.Text>
+          <Button
+            name="submit"
+            disabled={loading}
+            className="mt-2"
+            onClick={handleSubmit}
+          >
+            {t('Add')}
+          </Button>
+        </Form.Group>
+        <hr />
+        <Form.Group className="mb-3">
+          <Form.Label htmlFor="url">Remove a plugin</Form.Label>
+          <Form.Control
+            id="plugin-select"
+            as="select"
+            name="plugin"
+            type="text"
```

Comment:
This might be another leftover from copy-paste? You can remove the `type` attribute here, since it's only relevant for an `input` (and, since you're using `as="select"`, this form field is rendered as a `select`, not an `input`).

Reply:
Good catch!